frontend/docs/admin-guide/install.md

```diff
@@ -969,6 +969,232 @@ policy. From the IAM Service page click "Policies" on the left hand side.
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance-lifecycle-config/*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "Null": {
+                    "aws:ResourceTag/project": "false",
+                    "aws:ResourceTag/system": "false",
+                    "aws:ResourceTag/user": "false"
+                }
+            },
+            "Action": [
+                "sagemaker:DeleteNotebookInstance",
+                "sagemaker:DescribeNotebookInstance",
+                "sagemaker:StopNotebookInstance"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:DeleteNotebookInstanceLifecycleConfig",
+                "sagemaker:DescribeNotebookInstanceLifecycleConfig"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:sagemaker:{AWS_REGION}:{AWS_ACCOUNT}:notebook-instance-lifecycle-config/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "sagemaker:ListNotebookInstanceLifecycleConfigs",
+                "sagemaker:ListNotebookInstances"
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": "lambda:InvokeFunction",
+            "Resource": "arn:{AWS_PARTITION}:lambda:{AWS_REGION}:{AWS_ACCOUNT}:function:mls-lambda-*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:ListClusters",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "elasticmapreduce:DescribeCluster",
+                "elasticmapreduce:ListInstances",
+                "elasticmapreduce:TerminateJobFlows",
+                "elasticmapreduce:SetTerminationProtection"
+            ],
+            "Resource": "arn:{AWS_PARTITION}:elasticmapreduce:{AWS_REGION}:{AWS_ACCOUNT}:cluster/*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "translate:StopTextTranslationJob",
+                "translate:ListTextTranslationJobs",
+                "translate:DescribeTextTranslationJob",
+            ],
+            "Resource": "*",
+            "Effect": "Allow"
+        },
+        {
+            "Condition": {
+                "StringEqualsIgnoreCase": {
+                    "iam:ResourceTag/system": "MLSpace"
+                }
+            },
+            "Action": [
+                "iam:DetachRolePolicy",
+                "iam:DeleteRole",
+                "iam:DeleteRolePolicy",
+            ],
+            "Resource": "arn:{AWS_PARTITION}:iam::{AWS_ACCOUNT}:role/MLSpace*",
+            "Effect": "Allow"
+        },
+        {
+            "Action": [
+                "iam:ListRoles",
```

Comment:
I guess we can have the project create and user create policy use the App role, but it feels like a weird line to not include creating roles here and to split the project and user lambdas between different roles. 

It feels like creating roles is something the System role should be able to do.

Reply:
It's out of scope for the documentation PR, but I'm open to the discussion of moving all permissions for project/user processes to the system role (and anything else that we think should we handled by a system role rather than an application role used by users).